synapseclient/core/utils.py

```diff
@@ -880,3 +880,8 @@ def require_param(param, name):
     if param is None:
         raise ValueError("%s parameter is required." % name)
 
+
+def snake_case(string):
+    """Convert the given string from CamelCase to snake_case"""
+    # https: // stackoverflow.com / a / 1176023
```

Comment:
Interesting, is this a contention for commenting stack overflow urls?

Reply:
Not sure how those extra spaces got in there, probably just a copy and paste artifact. Fixed.